packages/cli/src/main.rs

```diff
@@ -110,6 +78,41 @@ async fn main() -> anyhow::Result<()> {
 
             Ok(())
         }
-        _ => Err(anyhow::anyhow!(bin.unwrap_err())),
+        action => {
+            let bin = get_bin(args.bin);
```

Comment:
We can use `get_bin(args.bin)?` now that all of the commands in this branch need bin

Reply:
```rust
        action => {
            let bin = get_bin(args.bin)?;
            let _dioxus_config = DioxusConfig::load(Some(bin.clone()))
                       .map_err(|e| anyhow!("Failed to load Dioxus config because: {e}"))?
                       .unwrap_or_else(|| {
                           log::info!("You appear to be creating a Dioxus project from scratch; we will use the default config");
                           DioxusConfig::default()
                    });

            #[cfg(feature = "plugin")]
            PluginManager::init(_dioxus_config.plugin)
                .map_err(|e| anyhow!("🚫 Plugin system initialization failed: {e}"))?;

            match action {
                Build(opts) => opts
                    .build(Some(bin.clone()), None)
                    .map_err(|e| anyhow!("🚫 Building project failed: {}", e)),

                Clean(opts) => opts
                    .clean(Some(bin.clone()))
                    .map_err(|e| anyhow!("🚫 Cleaning project failed: {}", e)),

                Serve(opts) => opts
                    .serve(Some(bin.clone()))
                    .await
                    .map_err(|e| anyhow!("🚫 Serving project failed: {}", e)),

                Bundle(opts) => opts
                    .bundle(Some(bin.clone()))
                    .map_err(|e| anyhow!("🚫 Bundling project failed: {}", e)),

                _ => Err(anyhow::anyhow!(bin.unwrap_err())),
            }
```

Like this? although Idk what last arm will return, since there may exist no error till there, So, should I make it `Ok(())`

Reply:
Yes, that looks much nicer! The `_ =>` branch can be `unreachable!()`